LoRaEngine/modules/LoRaWanNetworkSrvModule/Logger/LogSink.cs

```diff
@@ -0,0 +1,291 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#nullable enable
+
+namespace LoRaWan
+{
+    using System;
+    using System.Buffers;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.IO;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Sockets;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Channels;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.Extensions.Primitives;
+
+    public interface ILogSink
```

Comment:
We seem to add quite a bit of code for something we won't use in the future. When we move to the ILogger, this seems to be throw away code. Why bother for this intermediate step?